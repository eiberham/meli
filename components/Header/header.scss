.header {
    grid-area: head;
    /* background-color: #fff159; */
    display: flex;
    justify-content: center;
    align-items: center;
}