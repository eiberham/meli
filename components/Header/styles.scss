.header {
    grid-area: head;
    background-color: #fff159;
    display: flex;
    justify-content: center;
    align-items: center;
    max-height: 6vh;
}