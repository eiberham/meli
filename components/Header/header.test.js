import React from 'react';
import renderer from 'react-test-renderer';
import Header from './index';

describe('Header', () => {
    it('matches snapshot', () => {
        const component = renderer.create(<Header />)
        .toJSON();
        expect(component).toMatchSnapshot();
    })
})