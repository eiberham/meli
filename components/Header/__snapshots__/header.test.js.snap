// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<header
  className="header"
>
  <div
    className="header-content"
  >
    <div
      className="logo"
    >
      <img
        alt="logo"
        onClick={[Function]}
        onMouseEnter={[Function]}
        src="/static/Logo_ML.png"
      />
    </div>
    <div
      className="search"
    >
      <form
        className="css-1vr8bhw"
        onSubmit={[Function]}
      >
        <div
          className="ui small fluid action input"
        >
          <input
            id="search"
            name="search"
            onChange={[Function]}
            placeholder="Nunca dejes de buscar ..."
            type="text"
            value=""
          />
          <button
            className="ui icon button"
            onClick={[Function]}
            placeholder=""
          >
            <i
              aria-hidden="true"
              className="search icon"
              onClick={[Function]}
            />
          </button>
        </div>
      </form>
    </div>
  </div>
</header>
`;
